README.md

```diff
@@ -4,70 +4,40 @@ This Grafana-agent-k8s Terraform module aims to deploy the [grafana-agent-k8s ch
 
 ## Getting Started
 
-### Install requirements
+### Prerequisites
 
 The following software and tools needs to be installed and should be running in the local environment.
 
 - `microk8s`
 - `juju 3.x`
 - `terrafom`
 
-Install Microk8s:
-
-```console
-sudo snap install microk8s --channel=1.27-strict/stable
-sudo usermod -a -G snap_microk8s $USER
-newgrp snap_microk8s
-```
+### Deploy Mongodb-k8s using Terraform
 
-Enable `storage` plugin for Microk8s:
+Make sure that `storage` plugin is enabled for Microk8s:
 
 ```console
 sudo microk8s enable hostpath-storage
 ```
 
-Install Juju:
-
-```console
-sudo snap install juju --channel=3.1/stable
-```
-
-Install Terraform:
-
-```console
-sudo snap install --classic terraform
-```
-
-### Bootstrap the Juju using Microk8s and create a model to deploy Terraform module
-
-Bootstrap Juju Controller:
-
-```console
-juju bootstrap microk8s
-```
-
 Add a Juju model:
 
 ```console
 juju add model <model-name>
 ```
```

Comment:
Doesn't the model get added by the terraform module?

Reply:
Only for the upper level bundle (sdcore-control-plane), the juju model will be created by Terraform.

Reply:
AMF, SMF, Grafana  .. vs will not create model. The model name will be passed as a parameter.  
If the Juju model creation is added in each individual module,  it does not work as expected.

Reply:
I would have thought otherwise, that the model should be idempotent and should have been safe to ensure it exists.  That's too bad, but I guess it also prevents the need to repeat the same model assertion in all our modules.